examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_view_offset_consistency.py

```diff
@@ -0,0 +1,188 @@
+                is_first = False
+                continue
+            coord_list.append(self.__process_line(line))
+        return (original_coord, coord_list)
+
+    def compute_distance_to_original(self, axis=None):
+        if axis is None:
+            # Compute l2
+            return [np.linalg.norm(self.original_coord - coord) for coord in self.coord_list]
+        else:
+            # compute distance in 1 dimension, but extends negative
+            return [coord[axis] - self.original_coord[axis] for coord in self.coord_list]
+
+    def get_num_events(self):
+        return len(self.coord_list)
+
+    def get_num_failed_events(self, tolerance=None):
+        if tolerance is not None:
+            print(f"Overwriting tolerance value as {tolerance}")
+            self.tolerance = tolerance
+        return sum(errs > self.tolerance for errs in self.err_l2)
+
+
+def print_pass_and_fail(allowed_fraction_of_failed_events=0.5):
+    """
+    Prints the number of events, how many events failed and the percentage
+    :param allowed_fraction_of_failed_events: Fraction of events that could "fail" before the test failed
+    :return: None
+    """
+    print(f"\n\nInformation regarding pass fail rate of the view offset test\n"
+          f" ID || The Number of Events  ||  The Number of Failed Events  ||  Percentage Error\n"
+          f"-----------------------------------------------------------------------------------")
+    for key in point_sources_data.keys():
+        num_events = point_sources_data[key].get_num_events()
+        num_failed_events = point_sources_data[key].get_num_failed_events()
+        percentage = num_failed_events / num_events * 100
+        if percentage > allowed_fraction_of_failed_events * 100:
+            warning_msg = "HIGH VALUE WARNING!"
+        else:
+            warning_msg = ""
+        print(f"{key}  ||           {num_events}         "
+              f"||              {num_failed_events}                "
+              f"||      {round_sig(percentage, 3)}      {warning_msg}")
+
+
+def print_axis_biases():
+    """
+    Print the mean offset in each axis (x,y,z) for each point source file and the total bias in each axix
+    :return: None
+    """
+    total_bias_x = 0
+    total_bias_y = 0
+    total_bias_z = 0
+    print(f"\nMean offset in each axis for each source position\n"
+          f" ID ||   x   ||   y   ||   z\n"
+          f"------------------------")
+
+    for key in point_sources_data.keys():
+        mean_err_x = np.mean(point_sources_data[key].err_x)
+        mean_err_y = np.mean(point_sources_data[key].err_y)
+        mean_err_z = np.mean(point_sources_data[key].err_z)
+        total_bias_x += mean_err_x
+        total_bias_y += mean_err_y
+        total_bias_z += mean_err_z
+        print(f"{key}  "
+              f"|| {round_sig(mean_err_x)}  "
+              f"|| {round_sig(mean_err_y)}  "
+              f"|| {round_sig(mean_err_z)}  ")
```

Comment:
if you happen to know how to put consistent width in printing, precision and sign in Python, would be good to add it here